lib/tasks/orders.rake

```diff
@@ -0,0 +1,61 @@
+require 'rake'
+
+namespace :order do
+  desc "List Details about one or more Orders"
+  task :list => :environment do
+
+    with_tenant do
+      orders = Order.order('id DESC')
+
+      limit = ENV['LIMIT'].to_i
+      limit = 1 if limit.zero?
+
+      ids = ENV['ID'] || ENV['IDS']
+      if ids.present?
+        ids = ids.split(",")
+        orders = orders.where(:id => ids)
+        limit = ids.count
+      end
+
+      orders = orders.where(:owner => ENV['OWNER']) if ENV['OWNER'].present?
+
+      puts "Total Order items available - #{orders.count}"
+      orders = orders.limit(limit)
+
+      print_orders(orders)
+    end
+  end
+
+  def print_orders(orders)
+    orders.each do |order|
+      puts "\nORDER #{order.id}"
+      puts order.attributes.to_s
+
+      print_order_items(order.order_items)
+    end
+  end
+
+  def print_order_items(order_items)
+    order_items.each do |order_item|
+      puts "  ORDER ITEM #{order_item.id}"
+      puts "  #{order_item.attributes.inspect}"
+
+      print_progress_messages(order_item.progress_messages)
+    end
+  end
+
+  def print_progress_messages(progress_messages)
+    puts "    Progress Messages:"
+    progress_messages.order('received_at ASC').each do |pm|
+      puts "    #{pm.attributes.except(*%w(id created_at updated_at))}"
```

Comment:
@gmcculloug Should this just be printing the time, level and message.

Reply:
I wasn't sure how much to limit or expose and think I had those 3 before I switched to the current format.  Let me switch it back.

I see an opportunity to improve how all of this is logged, but wanted to get something started as most of the work was focused on the filtering options.